sympy/simplify/radsimp.py

```diff
@@ -1089,11 +1089,11 @@ def fraction(expr, exact=False):
                     denom.append(b)
                 elif exact:
                     if ex.is_constant():
-                        denom.append(Pow(b, -ex))
+                        denom.append(Pow(b, -ex, evaluate=not exact))
```

Comment:
Here exact is True so evaluate can be False.

Reply:
@oscargus thanks